File: Baekjoon/소수 판정/1978. 소수 찾기/소수 찾기.py
N = int(input())
answer = 0
number = list(map(int, input().split(' ')))
for i in number:
    count = 0
    for j in range(1, i + 1):
        if i % j == 0:
            count += 1
    if count == 2:
        answer += 1
print(answer)

File: Baekjoon/소수 판정/1978. 소수 찾기/소수 찾기.swift
import Foundation

var N = Int(readLine()!)!
var input = readLine()!.split(separator: " ").map { Int($0)! }
var ans = 0

for i in input {
    var cnt = 0
    for j in 1...i {
        if i % j == 0 {
            cnt += 1
        }
    }
    if cnt == 2 {
        ans += 1
    }
}

print(ans)